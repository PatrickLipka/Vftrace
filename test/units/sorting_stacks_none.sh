#!/bin/bash
set -x
test_name=sorting_stacks
output_file=${test_name}_none.out
ref_file=${srcdir}/ref_output/${test_name}_none.out

rm -f ${output_file}

export VFTR_SORT_PROFILE_TABLE="NONE"
if [ "x$HAS_MPI" == "xYES" ]; then
   ${MPI_EXEC} ${MPI_OPTS} ${NP} 1 ./${test_name} > ${output_file} || exit 1
else
   ./${test_name} > ${output_file} || exit 1
fi

diff ${output_file} ${ref_file}
