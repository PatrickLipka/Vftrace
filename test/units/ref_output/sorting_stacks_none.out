0(g4): init
   Thread: 0 (0) Overhead:        0, calls: 1, time(incl/excl): 20/0
1(g3): func0<init
   Thread: 0 (0) Overhead:        2, calls: 1, time(incl/excl): 10/7
   Thread: 1 (1) Overhead:        4, calls: 1, time(incl/excl): 10/6
   Thread: 2 (2) Overhead:        8, calls: 1, time(incl/excl): 10/8
   Thread: 3 (3) Overhead:       16, calls: 1, time(incl/excl): 10/5
   Thread: 4 (4) Overhead:       32, calls: 1, time(incl/excl): 10/7
   Thread: 5 (5) Overhead:       64, calls: 1, time(incl/excl): 10/4
2(g5): func1<init
   Thread: 0 (0) Overhead:      128, calls: 1, time(incl/excl): 10/10
   Thread: 1 (1) Overhead:      256, calls: 1, time(incl/excl): 10/7
   Thread: 2 (2) Overhead:      515, calls: 1, time(incl/excl): 10/6
3(g1): func2<func1<init
   Thread: 0 (1) Overhead:     1024, calls: 1, time(incl/excl): 3/3
   Thread: 1 (2) Overhead:     2048, calls: 1, time(incl/excl): 4/4
4(g6): func3<func0<init
   Thread: 0 (0) Overhead:     4096, calls: 1, time(incl/excl): 3/3
   Thread: 1 (2) Overhead:     8192, calls: 1, time(incl/excl): 2/2
   Thread: 2 (4) Overhead:    16384, calls: 1, time(incl/excl): 3/3
5(g2): func4<func0<init
   Thread: 0 (1) Overhead:    32768, calls: 1, time(incl/excl): 4/4
   Thread: 1 (3) Overhead:    65536, calls: 1, time(incl/excl): 5/5
   Thread: 2 (5) Overhead:   131072, calls: 1, time(incl/excl): 6/4
6(g0): func5<func4<func0<init
   Thread: 0 (5) Overhead:   262144, calls: 1, time(incl/excl): 2/2
